src/initializers/datadog/index.ts

```diff
@@ -0,0 +1,19 @@
+import requireInjected from '../../require-injected';
+import { getLogger } from '../log4js';
+
+let tracer;
+export default config => {
+  if (process.env.DD_SERVICE && process.env.DD_ENV) {
+    tracer = requireInjected('dd-trace').init();
+    tracer.use('koa', {
```

Comment:
maybe we can have an optional config with an array of datadog plugins to be enabled?

Reply:
According to the API found [here](https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md)  the instrumentation of external libraries using the built-in plugins is done automatically. You can also configure disabled plugins through env variables which is the recommended way to configure datadog, that's why I kept the initialisation simple.